backend/utils/permissions.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework.permissions import BasePermission
+
+class CustomBasePermission(BasePermission):
+    def has_permission(self, request, view):
+        if request.method in ['GET']:
```

Comment:
It is not always the case that if its a GET operation, then the user just needs to be authenticated, like for the users list. I'm wondering if it would make sense for you to combine the use of the `isAdminUser` and this custom permission or not. If you've already considered this, then disregard